### ISSUE TITLE
This can get me red trust factor?

### ISSUE THREAD

**Issue by undergroundhobo** [b1scoito/cozinha_loader#21]
Sorry for bothering, i'm really new to cheats and everything they can bring me.

I guess the title is self explanatory, but lol. This injector can get me a perma red trust factor?

**Comment by undergroundhobo** [comment]
Btw, I use Osiris, but only for inventory changer

**Comment by b1scoito** [comment]
Apparently the VAC3 bypass does. I cannot confirm it but it is highly possible that from my research once you inject the VAC3 bypass on Steam and start blocking modules, VAC3 can detect that behavior and give you red trust factor. As my loader uses the latest release on the [vac3_inhibitor](https://github.com/zyhp/vac3_inhibitor) repository, it's also possible that VAC3 created an unique signature for this specific release and is giving red trust factor for every account that you inject it on.

TL;DR: There's a possibility that VAC is detecting either the current public release by its signature, or it detects the modules getting blocked.

**Comment by undergroundhobo** [comment]
If I get a red trust factor by this, there is anyway to get it back to green? Or it's just a lost account to perma red?

Btw, thanks for the quick and polite response. English is not my main language, but I try hahaha

**Comment by b1scoito** [comment]
Most likely permanent to red.

**Comment by b1scoito** [comment]
I pushed a new update where the sig of the vac bypass itself changed. Worth some testing, nothing guaranteed though.

**Comment by undergroundhobo** [comment]
Nice one!

In a few days I'm getting a new notebook, and I'll give it a try on it
Thx a lot!

**Comment by 2lag** [comment]
If you are able to obfuscate and make your dll completely undetected with junk code among the obfuscation, why would you need a VAC bypass in the first place? Why could you not just inject into the game and keep your trust factor green??

**Comment by b1scoito** [comment]
Adding junk code / static obfuscation to your DLL doesn't mean it'll be undetected. It'll just not be detected by sig scanning, but depending on what you inject in your game, there might still be a detected hooking method or something along those lines still.

I added static obfuscation to the vac bypass DLL to see if vac was detecting my DLL by sig scanning the public vac bypass DLL

**Comment by 2lag** [comment]
Yes, you're absolutely right. In this case, I'm talking exclusively about CS:GO. I just see so many conflicting things on the internet, so many people push against using a VAC bypass because it is so easily detectable since most of the time you are blocking actual steam processes/functionality (I believe).
So I guess my question is, why would you want a VAC bypass if you've already made your cheat undetected to the method in which the AC will detect it? Why not just restart steam and load the DLL once it detects the CSGO process like every big P2C?

**Comment by b1scoito** [comment]
If you think the cheat you're using is already undetected by VAC3 you can absolutely use it without the VAC3 bypass.

The VAC3 bypass hooks on the loading module function and returns null on the runfunc call or deems the module as successfully loaded.

Not all famous P2C are fully undetected, some even use our method or similar of the vac3 bypass.

**Comment by 2lag** [comment]
O.o Okay, now that, I did not know. Thank you so much for the insight!

**Comment by MrCowboy20** [comment]
@b1scoito Plz help:
Did vac3 bypass binary compiled with LLVM-Obfuscator fix red trust factor?

**Thanks <3**

**Comment by b1scoito** [comment]
> @b1scoito Plz help: Did vac3 bypass binary compiled with LLVM-Obfuscator fix red trust factor?
> 
> **Thanks <3**

I can't tell you for sure, it's up to you to test it and see if it works as intended for you.

**Comment by xXxNozz4rrxXx** [comment]
> If you are able to obfuscate and make your dll completely undetected with junk code among the obfuscation, why would you need a VAC bypass in the first place? Why could you not just inject into the game and keep your trust factor green??

junk code??? lol

**Comment by ign54** [comment]
Does this sexy smexy injector have a virus, or files are false trusted?

**Comment by b1scoito** [comment]
> Does this sexy smexy injector have a virus, or files are false trusted?

Project is open-source, all code can be compiled by yourself. The executable has been compiled directly from the code and posted as release. Most likely false-flag from the anti-virus software since we call Windows API functions that are considered "suspect" by the anti-virus engine.

**Comment by xXxNozz4rrxXx** [comment]
this injector can give you red trust factor because as the @b1scoito  said every check return to null , and vac can detect this also vac cant ban you because he cant actually detect your paste shit or even your p2c, by the scans VAC knows something is not right so it will give you a red trust factor
